docs/articles/documentation/using-testcafe/common-concepts/reporters.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+title: Reporters
+permalink: /documentation/using-testcafe/common-concepts/reporters
+---
+# Reporters
+
+Reporters are plugins used to output test run reports in a certain format.
+
+TestCafe ships with the following reporters:
+
+* [spec](https://github.com/DevExpress/testcafe-reporter-spec) - used by default
+* [list](https://github.com/DevExpress/testcafe-reporter-list)
+* [minimal](https://github.com/DevExpress/testcafe-reporter-minimal)
+* [xUnit](https://github.com/DevExpress/testcafe-reporter-xunit)
+* [JSON](https://github.com/DevExpress/testcafe-reporter-json)
+
+You can also create a [custom reporter](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/) that will fit your needs.
+
+For more information about reporters, see the following sections.
+
+* [Searching for Reporter Plugins](#searching-for-reporter-plugins)
+* [Installing Reporters](#installing-reporters)
+* [Using Reporters](#using-reporters)
+
+## Searching for Reporter Plugins
+
+Reporter plugins are npm packages. A reporter package name consists of two parts - the `{testcafe-reporter-}` prefix and the name of a reporter itself, for example, `{testcafe-reporter-list}`.
+
+You can find available reporter packages by using the `npm search` command.
```

Comment:
`npm search` is quite slow and requires local npm index building. I'd rather give a link to online version: https://www.npmjs.com/search?q=testcafe-reporter